src/hotspot/share/logging/logAsyncWriter.cpp

```diff
@@ -102,7 +102,7 @@ class AsyncLogMapIterator {
     using none = LogTagSetMapping<LogTag::__NO_TAG>;
 
     if (*counter > 0) {
-      LogDecorations decorations(LogLevel::Warning, none::tagset(), output->decorators());
+      LogDecorations decorations(LogLevel::Warning, none::tagset(), LogDecorators::All);
```

Comment:
This looks odd to use LogDecorators::All - what exactly will that produce?

Reply:
This is the out-of-band message, it just shows that "xyz messages dropped due to async logging." 

Unlike normal log messages, this ad-hoc message is enqueued when buffer is flushing. The synchronization I described in `LogConfiguration::configure_output` only protects enqueuing log messages.  This message is not under protection.Without LogDecorators::All, it will fail 'gtest:LogConfigurationTest.reconfigure_decorators_MT*' in async log mode.